packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -102,51 +103,90 @@ export interface PhoneNumberInputProps
    */
   locale?: string | string[];
   /**
-   * TODO: Write a description
+   * Country code selector details.
    */
   countryCode: {
     /**
      * Visually hidden label for visually-impaired users.
      */
     label: string;
     /**
-     * TODO: Write a description. See https://en.wikipedia.org/wiki/List_of_country_calling_codes
+     * List of country calling codes to be rendered inside the selector
      */
     options: {
       /**
-       * TODO: Write a description
+       * Country name in two letter ISO 3166 region code format(https://www.iso.org/iso-3166-country-codes.html)
        */
       country: string;
       /**
-       * TODO: Write a description
+       * Country calling codes, see https://en.wikipedia.org/wiki/List_of_country_calling_codes
        */
       code: string;
     }[];
+    /**
+     * Triggers error styles on the component. Important for accessibility.
+     */
+    invalid?: boolean;
     /**
      * Initial country code.
      */
     defaultValue?: string;
     /**
-     * TODO: Write a description.
-     * TODO: The HTML `select` element doesn't support the `readonly` attribute, so this will need to be implemented using CSS and the `aria-readonly` and/or `disabled` attributes. The element must remain perceivable by screen reader users.
+     * Triggers readonly styles on the component.
      */
     readonly?: boolean;
+    /**
+     * Callback when the country code changes.
+     * The callback receives the normalized phone number in the [E.164 format](https://en.wikipedia.org/wiki/E.164),
+     * e.g. `+17024181234`.
+     */
```

Comment:
This is inaccurate. This `onChange` callback is specific to the country code `select` element and receives the change `event` (which only contains the country code as the `event.target.value`).